### ISSUE TITLE
"Body" or "authority"?

### ISSUE THREAD

**Issue by WilliamWDTK** [mysociety/whatdotheyknow-theme#1345]
I suspect for Alaveteli itself, they'll always be bodies, but otherwise, what are they?

It doesn't quite seem right to me to call a school or a hospital an "authority".  How are they referred to in law?

**Comment by confirmordeny** [comment]
In the Freedom of Information Act 2000, the term "public authority" is used. For example, in section 1: "Any person making a request for information to a public authority is entitled [...]". A public authority can be natural person such as a GP. The term body suggests that the entity is incorporated as a separate legal person. EIR also uses "public authority".

I believe that we use "body" in the code but "public authority" everywhere else.

**Comment by RichardTaylor** [comment]
We have "body" in URLs but "authority" in text. 

"Organisation" is another term we use. 

Note we list some companies in respect of eg. being a pharmacy. 

Do we have text on the site where this is a problem?

**Comment by RichardTaylor** [comment]
Change eg. 

>Found 25669 public authorities in the category ‘Schools’

to

>Found 25669 organisations in the category ‘Schools’

**Comment by confirmordeny** [comment]
I think "public authority" is better in the case of privately owned businesses performing public functions. For example, I would prefer call Boots a public authority than a public body.

**Comment by WilliamWDTK** [comment]
Calling Boots an authority sounds odd to me, really, since it isn't what one would call an "authority".  I think the main problem here is that there is not one word which covers all `bodies`.  "Body" is perhaps the most neutral, but it's still not accessible, and it was accessibility/comprehensibility that I was aiming for.

**Comment by WilliamWDTK** [comment]
@garethrees did mention on the call that he might look back at when/why it was changed with regard to the mismatch between code and general public text.
